dcat/index.html

```diff
@@ -2103,7 +2099,7 @@ <h3>Class: Data Service</h3>
         <table class="definition">
             <thead><tr><th>RDF Class:</th><th><a href="http://www.w3.org/ns/dcat#DataService">dcat:DataService</a></th></tr></thead>
             <tbody>
-            <tr><td class="prop">Definition:</td><td>A site or end-point providing operations related to the discovery of, access to, or processing functions on, data or related resources</td></tr>
+            <tr><td class="prop">Definition:</td><td>A collection of operations accessible through an application programming interface (API) that provide access to one or more datasets or data processing functions. </td></tr>
```

Comment:
However, I don't quite understand what the phrase "accessible through an application programming interface (API)" adds or changes relative to "end-point". Is it perhaps just that there is competing jargon/terminology coming from different domains?